docs/emulation.md

```diff
@@ -63,7 +63,8 @@ with a :gear:. You can get them
 - [Parallel Launcher](https://parallel-launcher.ca) - Nintendo 64 games
 - [:gear: DuckStation](https://www.duckstation.org) - PlayStation 1 games
 - [bsnes](https://github.com/bsnes-emu/bsnes) /
-  [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
+- [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
```

Comment:
This line break might be an accident.

Reply:
this is adding `-` in the last item.

Reply:
I mean, take a look at the line with melonDS and DeSmuME and you will see what's wrong with it. the `-` should be added to the new line, not to the unrelated Snes9x.